### PR TITLE
RNs-4.9.45 Added with bug fix and security update

### PR DESCRIPTION
**Pull request by Wiharris**
**RNs-4.9.45**
Added with bug fix and security update

**Version(s):**
* PR applies only to enterprise-4.9

**Issue:**
https://issues.redhat.com/browse/ART-4369

**Link to docs preview:**
https://wiharris.github.io/openshift-docs/RNs-4.9.45/release_notes/ocp-4-9-release-notes.html#ocp-4-9-45

**Additional information:**
`shipped live`

### CODE REVIEW

File: release_notes/ocp-4-9-release-notes.adoc [hunk 1]
```diff
@@ -3005,3 +3005,24 @@ link:https://access.redhat.com/solutions/6968056[{product-title} 4.9.43 containe
...
+{product-title} release 4.9.45, which includes security updates, is now available. The bug fixes that are included in the update are listed in the link:https://access.redhat.com/errata/RHSA-2022:5879[RHSA-2022:5879] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2022:5878[RHBA-2022:5878] advisory.
+
+Space precluded documenting all of the container images for this release in the advisory. See the following article for notes on the container images in this release:
+
+link:https://access.redhat.com/solutions/6970809[{product-title} 4.9.45 container image list]
+
+[id="ocp-4-9-45-bug-fixes"]
+==== Bug fixes
+
+* Previously, the metrics endpoints for Cinder CSI Driver and Cluster CSI Driver were using Transport Layer Security (TLS) configurations that were not secure. This caused attackers to gain significant access to vulnerable cryptography where they can decrypt and modify the endpoints. This fix improves the TLS configurations, resulting in the metrics endpoints being more secure from vulnerable cryptography. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2110255#[*BZ#2110255*])
```
Comment (JoeAldinger) [inline release_notes/ocp-4-9-release-notes.adoc:3023]:
```suggestion
* Previously, the metrics endpoints for Cinder CSI Driver and Cluster CSI Driver used unsecured Transport Layer Security (TLS) configurations. This allowed attackers access to vulnerable cryptography where they can decrypt and modify the endpoints. With this update, the TLS configurations are more secure and eliminate access to vulnerable cryptography. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2110255#[*BZ#2110255*])
```

Comment (Wiharris) [reply release_notes/ocp-4-9-release-notes.adoc]: done! :)

File: release_notes/ocp-4-9-release-notes.adoc [hunk 2]
```diff
@@ -3005,3 +3005,24 @@ link:https://access.redhat.com/solutions/6968056[{product-title} 4.9.43 containe
...
+{product-title} release 4.9.45, which includes security updates, is now available. The bug fixes that are included in the update are listed in the link:https://access.redhat.com/errata/RHSA-2022:5879[RHSA-2022:5879] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2022:5878[RHBA-2022:5878] advisory.
+
+Space precluded documenting all of the container images for this release in the advisory. See the following article for notes on the container images in this release:
+
+link:https://access.redhat.com/solutions/6970809[{product-title} 4.9.45 container image list]
+
+[id="ocp-4-9-45-bug-fixes"]
+==== Bug fixes
+
+* Previously, the metrics endpoints for Cinder CSI Driver and Cluster CSI Driver used unsecured Transport Layer Security (TLS) configurations. This allowed attackers access to vulnerable cryptography where they can decrypt and modify the endpoints. With this update, the TLS configurations are more secure and eliminate access to vulnerable cryptography. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2110255#[*BZ#2110255*])
```
Comment (kcarmichael08) [inline release_notes/ocp-4-9-release-notes.adoc:3023]: I didn't see it on the official product names list but it looks like Cinder CSI might have OpenStack as part of the name - might verify if possible.

Comment (kcarmichael08) [inline release_notes/ocp-4-9-release-notes.adoc:3023]: "This allowed attackers access" - to make clearer, I suggest "These configurations allowed..." or something to indicate what "this" is referring to.

**Review by kcarmichael08 — COMMENTED** [review]
Two small comments, otherwise LGTM!

File: release_notes/ocp-4-9-release-notes.adoc (hunk 2 above)
Comment (Wiharris) [reply release_notes/ocp-4-9-release-notes.adoc]: Hi @kcarmichael08, 

I did not find an official product name either; however, I did see where OpenStack was a part of the name in the documentation. I have made the above updates! :)

Comment (Wiharris) [reply release_notes/ocp-4-9-release-notes.adoc]: Done! :) Thanks for your help!

Comment (mburke5678) [reply release_notes/ocp-4-9-release-notes.adoc]: > access to vulnerable cryptography where they can decrypt and modify the endpoints

I'm not sure this is right. The original docs text says the attacker can decrypt or modify the **traffic** to and from these endpoints, not the endpoints themselves.

**Comment by mburke5678** [comment]
@Wiharris One comment. Otherwise LGTM

File: release_notes/ocp-4-9-release-notes.adoc (hunk 2 above)
Comment (Wiharris) [reply release_notes/ocp-4-9-release-notes.adoc]: @mburke5678 You are def. right! Thanks for catching that! Updates have been made. :)

**Review by mburke5678 — APPROVED** [review]

**Pull request merged**